packages/wonder-blocks-toolbar/components/toolbar.md

```diff
@@ -31,13 +31,15 @@ const rightContent = <React.Fragment>
 
 ### Small toolbar with multiple left side buttons:
 
+**Note:** The `IconButton`s are nudged to the left by 1px to match designs.
+
 ```js
 const {icons} = require("@khanacademy/wonder-blocks-icon");
 const Button = require("@khanacademy/wonder-blocks-button").default;
 const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
 const {Strut} = require("@khanacademy/wonder-blocks-layout");
 
-const zoomOutButton = <IconButton icon={icons.zoomOut} kind="primary"/>;
+const zoomOutButton = <IconButton icon={icons.zoomOut} kind="primary" style={{marginLeft:-1}}/>;
```

Comment:
Doesn't this feel wrong to you? I feel like either the designs need to be changed or the icons themselves need to be changed. Are we not centering the icons correctly or something? I think if we can't just drop a button or iconbutton into a toolbar and have it "work correctly" then we're failing somewhere! Maybe we can discuss this in the sync tomorrow?

Reply:
This is correct.  All of the icons are 24x24 but many of the visuals occupy some subset of that space.  In this case the `zoomOut` icon there's 1px padding on the left side.  In the case of the `dismiss` icon, the icon itself is only 12x12 so there's 6px of padding around it.  If we want IconButton to "just work" then we need to include additional information about it in the `IconAsset` data structure.  We should probably do that, but it feels like it's outside the scope of this PR.

Reply:
I'm not actually sure how we'd get this to "just work" since there are some situation where we want to use the visual size of the icon (aligning to the edge of the Toolbar) and other times we want to use the 24x24 bounding box (spacing between icons).  We should talk to design though about the spacing.  In the mocks we happen to use 24x24 bounding box between the zoom-in and zoom-out icons, but those icons are close enough to 24x24 that maybe the intention was to actually use the visual bounding box for that spacing.

Reply:
I'm interested in this too. I guess I'd feel more confident if the Zeplin mocks also included the focus state, since I'd know that the designer understood the visual implications of the focus rings. Right now I'd have to do a similar margin tweaking on my use of IconButton, and it feels a bit funny, as John said. Did you get a consensus on this in the design sync?

Reply:
There were no designers in the sync today.  I'll try to get an answer to this next week from design.  I think that this is a step in the right direction though.